pkg/platform/kube/platform.go

```diff
@@ -1038,3 +1051,14 @@ func (p *Platform) platformFunctionEventToFunctionEvent(platformFunctionEvent *p
 	functionEvent.Annotations = platformFunctionEvent.Meta.Annotations
 	functionEvent.Spec = platformFunctionEvent.Spec // deep copy instead?
 }
+
+func (p *Platform) validateAndEnrichAPIGatewayName(apiGateway *nuclioio.NuclioAPIGateway) error {
```

Comment:
```suggestion
func (p *Platform) enrichAndValidateAPIGatewayName(apiGateway *nuclioio.NuclioAPIGateway) error {
```

It may look silly but I think enrichment should be done first and if this is ever broken to 2 functions, enrichment should be called first or validations will fail